D2Go/README.md

```diff
@@ -0,0 +1,185 @@
+# D2Go on Android Using Native torchvision Ops Library
+
+## Introduction
+
+[Detectron2](https://github.com/facebookresearch/detectron2) is one of the most widely adopted open source projects and implements state-of-the-art object detection, semantic segmentation, panoptic segmentation, and human pose prediction. [D2Go](https://github.com/facebookresearch/d2go) is powered by PyTorch 1.8, torchvision 0.9, and Detectron2 with built-in SOTA networks for mobile.
+
+This D2Go Android demo app shows how to prepare and use the much lighter and faster D2Go model on Android. The code is based on a previous PyTorch Android [Object Detection demo app](https://github.com/pytorch/android-demo-app/tree/master/ObjectDetection) that uses a pre-trained Yolov5 model, with modified pre-processing and post-processing code required by the D2Go model.
+
+## Prerequisites
+
+* PyTorch 1.8.0 and torchvision 0.9.0
+* Python 3.8 or above
+* Android Pytorch library 1.8.0, torchvision library 1.8.0, torchvision_ops library 0.9.0
```

Comment:
nit: torchvision version should be 0.9.0, and not 1.8.0

Reply:
Actually in Android, the torchvision library is named as 1.8.0:
```
    implementation 'org.pytorch:pytorch_android:1.8.0-SNAPSHOT'
    implementation 'org.pytorch:pytorch_android_torchvision:1.8.0-SNAPSHOT'
    implementation 'org.pytorch:torchvision_ops:0.9.0-SNAPSHOT'
```

@IvanKobzarev can you please confirm?

Reply:
Yes, implementation 'org.pytorch:torchvision_ops:0.9.0-SNAPSHOT' is right
After publishing release versions we could remove SNAPSHOT part.

Reply:
@IvanKobzarev I think the confusion here was because we will now be providing a `pytorch_android_torchvision` artifact which is unrelated to torchvision.

Might be worth discussing separately about this to see if we can unify / remove one once we add the necessary functionality to torchvision